api-tests/test_update_responses.py

```diff
@@ -0,0 +1,62 @@
+import requests
+from io import StringIO
+from lxml import etree
+
+
+def test_update_addon_responses():
+    addon_urls = [
+        'https://aus5.mozilla.org/update/3/SystemAddons/44.0/20160310153207/Darwin_x86_64-gcc3-u-i386-x86_64/en-GB/release/Darwin%2014.5.0/default/default/update.xml',
```

Comment:
The tests should be run using 'py.test  api-tests/' which you should be able to do via CI. I will do up some longer instructions.

We don't have to hardcode the server at all, can definitely be passed in. Our choices are via the CLI or in a configuration file grouped by environment and then we pass in the environment as a CLI switch.